views/widgets/ctrl.php

```diff
@@ -26,9 +26,9 @@ function make_button( $label, $action, $referer = false, $level = 'low') {
 			<p><strong><?php esc_html_e('These actions affect all cached opcodes.' ,'opcache'); ?></strong>
 			<p>
 				<?php printf(
-					esc_html__('Please refer to %s for these difference information.', 'opcache'),
+					esc_html__('Please refer to %s for information about the different options.', 'opcache'),
 					sprintf('<a href="%1$s" target="_blank">%2$s</a>',
-						esc_url( OPcache_dashboard::PHP_URL . '/ref.opcache'),
+						esc_url( OPcache_dashboard::PHP_URL . '/book.opcache.php'),
```

Comment:
I prefer http://php.net/ref.opcache. Because I think what users want to know is the differences between "invalidate" and "reset", not setup instructions.